domains/devendra.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devendrakanojiya",
+        "email": "s_36100@bkbirlacollegekalyan.com"
+    },
+    "record": {
+        "CNAME": "devendrakanojiya.github.io/port"
```

Comment:
Invalid CNAME record (cannot contain `/`)

Reply:
Check now, resolved it 🙂